piqe_ocp_lib/api/crd/local_storage_operator.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from typing import Optional
+
+from kubernetes.client.rest import ApiException
+from openshift.dynamic.resource import ResourceInstance
+
+from piqe_ocp_lib import __loggername__
+from piqe_ocp_lib.api.resources.ocp_base import OcpBase
+from piqe_ocp_lib.api.resources.ocp_operators import ClusterServiceVersion
+from piqe_ocp_lib.api.tasks.operator_ops import OperatorInstaller
+
+logger = logging.getLogger(__loggername__)
+
+
+class LocalStorageOperator(OcpBase):
+    """
+    LocalStorageOperator Class extends OcpBase
+    related to local storage operator
+    :param kube_config_file: A kubernetes config file.
+    :return: None
+    """
+
+    def __init__(self, kube_config_file=None):
+        super().__init__(kube_config_file=kube_config_file)
+        self.oi_obj = OperatorInstaller()
+        self.check_operator_install = self.oi_obj.check_operator_installed("local-storage-operator")
+        self.version = self.oi_obj.get_version_of_operator("local-storage-operator")
+        self.channel = self.oi_obj.get_channel_of_operator("local-storage-operator")
+
+
+class LocalVolume(LocalStorageOperator):
+    """
+    LocalVolume Class extends LocalStorageOperator
+    related to local volume
+    :param kube_config_file: A kubernetes config file.
+    :return: None
+    """
+
+    def __init__(self, kube_config_file=None):
+        super().__init__(kube_config_file=kube_config_file)
+        self.api_version = "local.storage.openshift.io/v1"
+        self.kind = "LocalVolume"
+        self.lv = self.dyn_client.resources.get(api_version=self.api_version, kind=self.kind)
+
+    def create_local_volume(self) -> Optional[ResourceInstance]:
+        """
+        create local volume
+        :return: api reponse
+        """
+        if self.check_operator_install is not None and self.version is not None and self.channel is not None:
```

Comment:
more nitpick :) you can make this long if statement more 'pythonic' as suggested in this link
https://newbedev.com/what-is-the-most-pythonic-way-to-check-if-multiple-variables-are-not-none